README.md

```diff
@@ -289,3 +290,18 @@ spotless {
     }
 }
 ```
+
+## com.palantir.baseline-reproducibility
+
+This plugin is a shorthand for the following snippet, which opts-in to reproducible behaviour for all Gradle's Jar, Tar and Zip tasks. (Surprisingly, these tasks are not reproducible by default).
+
+```gradle
+tasks.withType(AbstractArchiveTask) {
+    preserveFileTimestamps = false
+    reproducibleFileOrder = true
+}
+```
```

Comment:
See https://dzone.com/articles/reproducible-builds-in-java for where I discovered this!